apps/ima/behaviours/continue-report.js

```diff
@@ -1,7 +1,23 @@
 const config = require('../../../config');
+const NotifyClient = require('notifications-node-client').NotifyClient;
+const notifyApiKey = config.govukNotify.notifyApiKey;
+const notifyClient = new NotifyClient(notifyApiKey);
+const templateId = config.govukNotify.saveAndExitTemplateId;
+const tokenGenerator = require('../../../db/save-token');
 
 const DEFAULTS = config.sessionDefaults;
 
+const getPersonalisation = (host, token) => {
+  const protocol = host.includes('localhost') ? 'http' : 'https';
+  return {
+    // pass in `&` at the end in case there is another
+    // query e.g. ?hof-cookie-check
+    link: `${protocol}://${host + config.login.appPath}?token=${token}&`,
+    host: `${protocol}://${host}`
+  };
+};
+
```

Comment:
Should really import and use the same `getPersonalisation` function in `save-form-session.js` and `continue-report.js` if they are exactly the same. `/lib/utilities.js` could be a good spot unless theres a fair reason to keep both functions.

Reply:
great point - have moved this function to lib/utilities and imported it into the behaviours